chromium_src/third_party/blink/renderer/modules/webaudio/realtime_analyser.h

```diff
@@ -0,0 +1,11 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#define BRAVE_REALTIMEANALYSER_H                                             \
```

Comment:
so future me doesn't wonder why we needed this here, can you add a comment? I believe it was because we don't have access to the Document for `GetFudgeFactor`?

Reply:
yes, and added